debugtools/DDR_VM/data/superset-constants.dat

```diff
@@ -5030,6 +5030,7 @@ S|OmrcfgConstants|OmrcfgConstantsPointer|
 C|OMRTHREAD_LIB_UNIX
 C|OMR_ARCH_X86
 C|OMR_ENV_DATA64
+C|OMR_GC_SPARSE_HEAP_ALLOCATION
```

Comment:
Additions to this file are not permitted unless it can be demonstrated that the constant has always existed (not the case here as `OMR_GC_SPARSE_HEAP_ALLOCATION` is introduced by https://github.com/eclipse/omr/pull/6948 which hasn't yet been merged).